operators/pkg/controller/remotecluster/driver_incluster.go

```diff
@@ -143,12 +154,18 @@ func doReconcile(
 		return updateStatusWithPhase(&remoteCluster, v1alpha1.RemoteClusterFailed), err
 	}
 
+	// Create remote service for seeding
+	svc, err := reconcileRemoteService(r.Client, r.scheme, remoteCluster)
```

Comment:
Nit: I would appreciate a bit of consistency between the two function names `seedServiceFinalizer` and `reconcileRemoteService`.